portal/messages/es.json

```
@@ -255,16 +255,17 @@
       "use-or": "Utilice <min></min> o <max></max>",
       "years": "{years, plural, one {{years} año} other {{years} años}}"
     },
-    "heading": "Staking de Gobernanza",
+    "heading": "Stake {symbol} de Gobernanza",
     "here-is-your-stake-tx": "Aquí está su transacción de staking:",
     "lockup-period": "Período de bloqueo",
     "stake-successful": "Stake exitoso",
-    "sub-heading": "Aprenda y explore con el ecosistema para desarrolladores de Hemi",
+    "sub-heading": "Bloquee {symbol}, obtenga poder de voto y ayude a dar forma al protocolo.",
+    "switch-to-start-staking": "Puede cambiar de red para comenzar a hacer staking de su HEMI.",
     "table": {
       "connect-to-hemi": "Conéctese a {network} para ver sus posiciones de stake.",
-      "connect-to-stake": "Conecte su billetera para ver su HEMI en stake.",
-      "get-started": "Comience haciendo staking de sus HEMI",
-      "no-hemi-staked": "No hay HEMI stakeado",
+      "connect-to-stake": "Conecte su billetera para ver su {symbol} en stake.",
+      "get-started": "Comience haciendo staking de sus {symbol}",
+      "no-hemi-staked": "No hay {symbol} stakeado",
```

One extra suggestion:

```suggestion
      "no-hemi-staked": "No hay {symbol} en stake",
```

I think this reads better